lib/cs_guide_web/controllers/venue_controller.ex

```diff
@@ -255,4 +329,20 @@ defmodule CsGuideWeb.VenueController do
       v.venue_name != venue_name
     end)
   end
+
+  def sortImagesByMostRecent(venue) do
+    Map.update(venue, :venue_images, [], fn images ->
+      images
+      |> Enum.sort(fn img1, img2 ->
+        img1.id >= img2.id
+      end)
+    end)
+  end
+
+  defp getPhotoOfPhotoNumber(venue, photo_number) do
```

Comment:
I don't think I can, because if there was more than one of each number you might end up with more than one of each photo and then you need to ensure you're getting the most recent of each photo number (which is why before those lines of code it sorts by most recent)

Does that make sense or would your method be able to account for those needs?

Reply:
Oh I see what you mean. That makes sense then 👍